generators/client/templates/angular/src/main/webapp/app/_app.module.ts

```diff
@@ -1,3 +1,4 @@
+import './vendor.ts';
```

Comment:
If I don't import this, the style tags for bootstrap and fa aren't injected in index.html. It might be because we build them out of the html webpack process. I didnt find a better solution so if you have a better idea